eis_toolkit/statistical_analyses/statistical_testing.py

```diff
@@ -0,0 +1,85 @@
+import pandas as pd
+from beartype import beartype
+from beartype.typing import Literal, Optional
+from scipy.stats import anderson, chi2_contingency, shapiro
+
+from eis_toolkit import exceptions
+
+
+def _statistical_tests(
+    data: pd.DataFrame, target_column: str, method: Literal, min_periods: Optional[int], delta_degrees_of_freedom: int
+) -> dict:
+
+    statistics = {}
+    normality = {}
+
+    for column in data.columns:
+        if data[column].dtype != float:  # Categorical variables
```

Comment:
I don't think we can assume that this criterion works for all cases.

Reply:
You're correct. The problem is that integers can be both categorical and numerical variables. One way to solve this would be to give categorical and numerical data as two separate inputs. But then the data must somehow be split before the function call.
Another option is that the user specifies which variables (=columns in df) are considered categorical. Any thoughts?